support/cmd_builder/src/ps.rs

```diff
@@ -0,0 +1,307 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Powershell Command Builder
+
+use crate::CommandError;
+use crate::run_cmd;
+use std::ffi::OsStr;
+use std::ffi::OsString;
+use std::path::Path;
+use std::path::PathBuf;
+use std::process::Command;
+
+/// A PowerShell script builder
+pub struct PowerShellBuilder(Command);
+
+impl PowerShellBuilder {
+    /// Create a new PowerShell command
+    pub fn new() -> Self {
+        PowerShellCmdletBuilder(Command::new("powershell.exe"))
```

Comment:
If you're going to keep this hardcoded as `powershell.exe`, then the whole crate should probably have `#![cfg(windows)]` (and the Cargo.toml should probably scope the dependencies to Windows, too).